compiler/x/codegen/OMRCodeGenerator.cpp

```diff
@@ -98,11 +98,17 @@
 #include "x/codegen/X86Instruction.hpp"
 #include "codegen/InstOpCode.hpp"
 
+// Amount to be added to the estimated code size to ensure that there are long
+// branches between warm and cold code sections (must be multiple of 8 bytes).
+//
+#define MIN_DISTANCE_BETWEEN_WARM_AND_COLD_CODE 512
```

Comment:
How is "512" chosen?  Is this any need or benefit to vary this?

Reply:
According to the comment it's to ensure that branches between warm and cold code are long. Is it a correct assumption for Intel?